components/worktemplates/components/customize.tsx

```diff
@@ -0,0 +1,119 @@
+import {Visibility} from '@mattermost/types/worktemplates';
+import Input from 'components/widgets/inputs/input/input';
+
+export interface CustomizeProps {
+    className?: string;
+    name: string;
+    visibility: Visibility;
+
+    onNameChanged: (name: string) => void;
+    onVisibilityChanged: (visibility: Visibility) => void;
+}
+
+const Customize = ({
+    name,
+    visibility,
+    onNameChanged,
+    onVisibilityChanged,
+    ...props
+}: CustomizeProps) => {
+    const {formatMessage} = useIntl();
+
+    return (
+        <div className={props.className}>
+            <div className='name-section-container'>
+                <p>
+                    <strong>
+                        {formatMessage({id: 'worktemplates.customize.name_title', defaultMessage: 'What project is this for?'})}
+                    </strong>
+                </p>
+                <p className='customize-name-text'>
+                    {formatMessage({id: 'worktemplates.customize.name_description', defaultMessage: 'This will help you and others find your project items. You can always edit this later.'})}
+                </p>
+                <input
+                    type='text'
+                    autoFocus={true}
+                    placeholder={formatMessage({id: 'worktemplates.customize.name_input_placeholder', defaultMessage: 'e.g. Web app, Growth, etc.'})}
+                    value={name}
+                    onChange={(e) => onNameChanged(e.target.value)}
+                />
+
+                <Input
+                    label='Test'
+                    value={name}
+                    onChange={(e) => onNameChanged(e.target.value)}
+                    placeholder={formatMessage({id: 'worktemplates.customize.name_input_placeholder', defaultMessage: 'e.g. Web app, Growth, etc.'})}
+                />
+            </div>
+            <div className='visibility-section-container'>
+                <p>
+                    <strong>
+                        {formatMessage({id: 'worktemplates.customize.visibility_title', defaultMessage: 'Who should have access to this?'})}
+                    </strong>
+                </p>
+                <PublicPrivateSelector
+                    selected={visibility === Visibility.Public ? 'O' : 'P'}
+                    onChange={(selected) => onVisibilityChanged(selected === 'O' ? Visibility.Public : Visibility.Private)}
+                />
+            </div>
+        </div>
+    );
+};
+
+const StyledCustomized = styled(Customize)`
+    display: flex;
+    flex-direction: column;
+    width: 509px;
+    margin: 0 auto;
+
+    .public-private-selector {
+        flex-direction: column;
+        &-button {
+            margin-bottom: 8px;
+            &:not(:first-child) {
+                margin-left: 0px;
+            }
+        }
+    }
+
+    strong {
+       font-weight: 600;
+        font-size: 14px;
```

Comment:
nit: there is an error with the formatting here, but the linter is not able to catch this kind of problems when using styled components.  😢 .. my very very personal opinion/preference, keep using scss modules

Reply:
To your point, it might be worth checking https://styled-components.com/docs/tooling#stylelint to allow stylelint to parse styled components 🤔 I surely won't go through this with this PR, it is big enough as it is without fixing tons of random files when this is added haha! But this is something that could be explored.

Reply:
or we could stop using styled.components 🙈